Dockerfile.tools

```diff
@@ -72,14 +72,15 @@ ENV GOCACHE="/go/.cache"
 ENV PATH="${PATH}:/usr/local/go/bin"
 ENV GOPATH="/go"
 
-WORKDIR /home/test
-COPY internal/test/wmcb/templates templates
+WORKDIR /payload
 COPY internal/test/wmcb/powershell/ .
-
 COPY --from=build /build/ovn-kubernetes/go-controller/_output/go/bin/windows/hybrid-overlay-node.exe .
 COPY --from=build /build/kubernetes/_output/local/bin/windows/amd64/kubelet.exe .
 COPY --from=build /build/wmcb_unit_test.exe .
 COPY --from=build /build/wmcb_e2e_test.exe .
+
+WORKDIR /home/test
```

Comment:
Why do we need a separate directory for the template? We should definitely move away from `/home/test/` either way.